source/documentation/administration_guide/topics/The_oVirt_Engine_Rename_Tool.adoc

```diff
@@ -22,6 +22,27 @@ The `ovirt-engine-rename` command updates records of the fully qualified domain
 
 * /etc/pki/ovirt-engine/keys/apache.p12
 
+
+[NOTE]
+====
+Are you sure you need to do this?
+
+Since version 4.0.4, it is possible to add more names to access the {engine-name} web interface.
+
+. Make sure the names you choose can resolve to an IP address of the {engine-name} machine by adding relevant records to the DNS names or to `/etc/hosts`.
```

Comment:
Sorry, one more nit: IMO not "to the DNS names", but "to the DNS" (as a system - Domain Name System).  If you think this is confusing, perhaps something longer can work, e.g.:
- Decide what name or names you want to use for the web interface.
- Make sure you can resolve these names - e.g. with `ping enginename` or `getent hosts enginename` - from your machine, and/or any other machine that needs access.
- Make sure the names resolve to an IP address of the engine machine - often, its only address, or one of them (or more), if it has more than one address. E.g. that `ping enginename` indeed pings the engine machine.
- Common ways to do this are to add the name to the `/etc/hosts` file on the respective machines, or to add them to a DNS server that these machines use.

This might seem slightly too long and obvious, but we do sometimes get reports which can be handled easily by methodically going through some procedure like above.